### PR TITLE
c08 translation

### PR DESCRIPTION
**Pull request by ManlyMarco**
Thanks to @ScrewThisNoise for providing the translations.

### CODE REVIEW

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c08/00/1000/translation.txt [hunk 1]
```diff
@@ -10,7 +10,7 @@
 「うん。私って子供っぽく思われがちでしょ？」=“Yeah. I mean, I come off as a kid a whole lot, don't I?”
 「だから、一度でいいからアダルトな雰囲気の場所でデートしてみたかったんだぁ」=“So I've always wanted to have a mature adult date in a place like this, atleast once~”
 「背伸びをしてるのはわかってるけど、[P名]君にふさわしい大人な彼女さんになりたくて」=“I know it's a bit too much for little old me but, I just wanted to be a mature girl that's worthy of you, just the one time, [P]-kun.”
-「[H名]……」=“[H名]...”
+「[H名]……」=“[H]...”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c08/00/1000/translation.txt:13]:
```suggestion
「[H名]……」=“[H名]...”
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c08/00/5/translation.txt [hunk 2]
```diff
@@ -2,5 +2,5 @@
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
 「うん……エッチ、しよ？」=“Okay... let's have, sex?”
-「[H名]……」=“[H名]...”
+「[H名]……」=“[H]...”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c08/00/5/translation.txt:5]:
```suggestion
「[H名]……」=“[H名]...”
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c08/00/1008/translation.txt [hunk 3]
```diff
@@ -10,7 +10,7 @@
 「そう？　[H名]は可愛いからすごく目立つと思うけど」=“You think? You're ridiculously adorable so I think you would, personally.”
 「えへへ。ありがと♪」=“Eheheh, thanks♪”
 「でも、他にもたくさん可愛い子がいるから……。私なんかじゃ敵わないよ」=“But, there's lots of other cute girls here too... I can't compete with them.”
-「[H名]……」=“[H名]...”
+「[H名]……」=“[H]...”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c08/00/1008/translation.txt:13]:
```suggestion
「[H名]……」=“[H名]...”
```

File: Translation/en/RedirectedResources/assets/abdata/communication/info_31/communication_08/translation.txt [hunk 4]
```diff
@@ -7,41 +7,41 @@
 「……すーすー………んん～……すーすー……」=“...*snore*... nnn~... *snore*...”
 「あの～……ちょっとだけ恥ずかしいかな」=“Uhm~... this is a bit embarrassing.”
 「じっと見てどうしたの？」=“You're staring, you know? What's up?”
-「うん？　付いて来てるよ？」=“Yeah? I'm following you, okay?”
-「？　ここは男の子は入れない所だよ？」=“...? This a place where boys aren't allowed, okay?”
+「うん？　付いて来てるよ？」=“Yeah? I'm right here, you know?”
+「？　ここは男の子は入れない所だよ？」=“? Boys can't come in here, you know?”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/communication/info_31/communication_08/translation.txt:11]:
```suggestion
「？　ここは男の子は入れない所だよ？」=“Boys can't come in here, you know?”
```

or

```suggestion
「？　ここは男の子は入れない所だよ？」=“Huh? Boys can't come in here, you know?”
```

File: Translation/en/RedirectedResources/assets/abdata/communication/info_50/communication_08/translation.txt [hunk 5]
```diff
@@ -7,41 +7,41 @@
 「……すーすー………んん～……すーすー……」=“...*snore*... nnn~... *snore*...”
 「あの～……ちょっとだけ恥ずかしいかな」=“Uhm~... this is a bit embarrassing.”
 「じっと見てどうしたの？」=“You're staring, you know? What's up?”
-「うん？　付いて来てるよ？」=“Yeah? I'm following you, okay?”
-「？　ここは男の子は入れない所だよ？」=“...? This a place where boys aren't allowed, okay?”
+「うん？　付いて来てるよ？」=“Yeah? I'm right here, you know?”
+「？　ここは男の子は入れない所だよ？」=“? Boys can't come in here, you know?”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/communication/info_50/communication_08/translation.txt:11]: Same as above....

**Review by GeBo1 — CHANGES_REQUESTED** [review]

**Review by GeBo1 — APPROVED** [review]

File: Translation/en/RedirectedResources/assets/abdata/communication/info_00/communication_23/translation.txt [hunk 6]
```diff
@@ -298,7 +298,7 @@
 「……むむ、ファンとして見に行くべきか」=“...Mumuh, as a fan, guess I've no choice but to go watch it.”
 //「おススメの漫画、教えてよ」=
 //「お安い御用ですぞ。どんなジャンルがいいか言ってくれれば厳選するよ」=
-「好きだよ、[H]。[H]はどう、かな？」=“I love you, [H]。[H]. What, about you?”
+「好きだよ、[H]。[H]はどう、かな？」=“I love you, [H]。[H]。[H]。[H]. What, about you?”
```
Comment (ManlyMarco) [inline Translation/en/RedirectedResources/assets/abdata/communication/info_00/communication_23/translation.txt:301]:
```suggestion
「好きだよ、[H]。[H]はどう、かな？」=“I love you, [H]. What about you?”
```

File: Translation/en/RedirectedResources/assets/abdata/communication/info_31/communication_23/translation.txt [hunk 7]
```diff
@@ -298,7 +298,7 @@
 「……むむ、ファンとして見に行くべきか」=“...Mumuh, as a fan, guess I've no choice but to go watch it.”
 //「おススメの漫画、教えてよ」=
 //「お安い御用ですぞ。どんなジャンルがいいか言ってくれれば厳選するよ」=
-「好きだよ、[H]。[H]はどう、かな？」=“I love you, [H]。[H]. What, about you?”
+「好きだよ、[H]。[H]はどう、かな？」=“I love you, [H]。[H]。[H]。[H]. What, about you?”
```
Comment (ManlyMarco) [inline Translation/en/RedirectedResources/assets/abdata/communication/info_31/communication_23/translation.txt:301]:
```suggestion
「好きだよ、[H]。[H]はどう、かな？」=“I love you, [H]. What about you?”
```

**Pull request merged**